fix(SEO): Add SEO descriptions to English source docs (#4997)
QwenLM/qwen-code: docs/users/features/headless.md

---

> description: "Run Qwen Code without a UI using Headless Mode. Build AI-powered scripts, CI/CD pipelines, and automation tools — no terminal required."

Reviewer: "no terminal required" reads backwards — headless mode is precisely how you run Qwen Code *from* the terminal/command line; what it drops is the interactive UI.

```suggestion
description: "Run Qwen Code without the interactive UI using Headless Mode. Build AI-powered scripts, CI/CD pipelines, and automation tools from the command line."
```
